src/test/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerLinkCaseTest.java

```diff
@@ -29,6 +36,9 @@ class CaseWorkerLinkCaseTest {
     @InjectMocks
```

Comment:
Change class name to CaseworkerLinkCaseTest

Reply:
Done